package.json

```
@@ -0,0 +1,48 @@
+{
+  "name": "whitelist-smart-contract",
+  "version": "0.0.1",
+  "description": "Whitelist smart contract to allow developers to not resign a sell order every time there's a new web3mail dapp being deployed.",
+  "main": "index.ts",
+  "scripts": {
+    "compile": "hardhat compile",
+    "test": "hardhat test",
+    "coverage": "hardhat coverage",
+    "deploy": "hardhat run scripts/deploy.ts --network bellecour",
+    "addRessourceToWhitelist": "hardhat run scripts/addRessourceToWhitelist.ts --network bellecour",
+    "removeRessourceFromWhitelist": "hardhat run scripts/removeRessourceFromWhitelist.ts --network bellecour",
+    "transferOwnership": "hardhat run scripts/transferOwnership.ts --network bellecour",
+    "verify": "hardhat verify",
+    "clean": "hardhat clean",
+    "lint": "eslint **/*.ts",
+    "lint:fix": "eslint **/*.ts --fix",
+    "typecheck": "tsc --noEmit",
+    "format": "prettier --plugin=./node_modules/prettier-plugin-solidity/src/index.js --write \"{scripts,test,contracts}/**/*.{ts,sol}\"",
+    "check-format": "prettier --plugin=./node_modules/prettier-plugin-solidity/src/index.js --check \"{scripts,test,contracts}/**/*.{ts,sol}\""
+  },
+  "devDependencies": {
+    "@nomicfoundation/hardhat-toolbox": "^3.0.0",
+    "@types/chai": "^4.3.6",
+    "@types/mocha": "^10.0.2",
+    "@types/node": "^20.8.0",
+    "@typescript-eslint/eslint-plugin": "^6.7.4",
+    "@typescript-eslint/parser": "^6.7.4",
+    "chai": "^4.3.10",
+    "eslint": "^8.50.0",
+    "eslint-config-airbnb-base": "^15.0.0",
+    "eslint-config-airbnb-typescript": "^17.1.0",
+    "eslint-config-prettier": "^9.0.0",
+    "eslint-plugin-import": "^2.28.1",
+    "eslint-plugin-prettier": "^5.0.0",
+    "hardhat": "^2.14.1",
+    "hardhat-deploy": "^0.11.37",
+    "mocha": "^10.2.0",
+    "prettier": "^3.0.3",
+    "prettier-plugin-solidity": "^1.1.3",
+    "ts-node": "^10.9.1",
+    "typescript": "^5.2.2"
+  },
+  "dependencies": {
+    "@nomicfoundation/hardhat-chai-matchers": "^2.0.2",
+    "@openzeppelin/contracts": "^4.9.3"
+  }
```

```suggestion
  "devDependencies": {
    "@nomicfoundation/hardhat-toolbox": "^1.0.2",
    "@nomiclabs/hardhat-etherscan": "^3.1.7",
    "hardhat": "^2.14.1",
    "prettier": "^2.8.8",
    "prettier-plugin-solidity": "^1.1.3"
  },
  "dependencies": {
    "@openzeppelin/contracts": "^4.8.2"
  }
```
clean useless dependency

---

actually only mocha is not used the others are used to by eslint and test script